cdk/infrastructure/constructs/frontend.py

```diff
@@ -127,6 +127,8 @@ def _define_fargate_service(self) -> None:
     def _configure_health_check(self) -> None:
         self.fargate_service.target_group.configure_health_check(
             interval=Duration.seconds(60),
+            healthy_http_codes='200',
+            path='/search',
```

Comment:
Home page redirect to query page. Should check `path='/query'`

Reply:
done